charters/ExpressionMatrixService/charter.md

```diff
@@ -0,0 +1,93 @@
+
+# Expression Matrix Service
+
+## Description
+
+The expression matrix service enables researchers to easily access and analyze HCA expression matrix data.
+
+## Objective
+
+* Build, operate, and maintain a data access API that serves expression
+matrices from the HCA DCP Data Store with options to subset data based on bundle metadata, genes, and numerical filters.
+
+* Serve as a test bed for a re-usable metadata query service, created by
+the *Query Service* component.
+
+## In-scope
+
+* Expression Matrix Service interface supports queries based on:
+    * Bundle identifiers
+        * This query enables selection of cells from specific submissions. 
+    * Metadata (*across bundles; potentially provided directly by Query Module*)
+        * These types of filters will enable users to select cells based on their biological characteristics, for example, all kidney cells, or all cells prepared using the 10x v2 assay.
+    * Numerical filtering (*on gene expression values, and also QC metrics*)
+        * These types of filters will, for example, enable users to query for cells according to their expression of key marker genes, or exclude cells with low quality scores. 
+    * Compound metadata and numerical filtering (*combinations of the above filters*)
+
+* Expression Service API and scalable backend that provides access to
+query interface
+
+* Architecture, software design, and scale testing for scalability
+
+* Data representation
+    * Expression matrix data format for transport
+    * Expression matrix data format for storage
+    * QC matrix data format for storage and transport
+
+
+* Extends HCA DCP Command Line Interface (CLI) client to support the Expression Matrix Service
+
+* Documentation
+    * Publish documentation for API and file formats
+    * Publish example code for developers to enable fast/easy adoption of API by analysis and visualization portals
+
+* User Experience
+    * Conduct UX research and design; drive query and API design directly from use cases
+    * Validate design with communities building portals for analysis and
+    visualization
+    * Provide feedback in the form of architecture sessions and written UX
+    reports to the DCP community about the experience of building layered services using the public API(s). (e.g., *can the expression microservice be built using only public Data Store API?*)
+    * Scientific "guardrails" to warn users when executed queries produce data subsets with questionable scientific utility
+
+* External dependencies
+    * Dependencies on Data Store, Query Service, and Secondary Analysis to enable and expose quality control metrics
```

Comment:
Secondary Analysis also generates data matrices as well as QC metrics, not sure if you want to add that here in the dependencies.